rpm/libxml2.spec

```diff
@@ -88,22 +71,10 @@ install -m0644 -t $RPM_BUILD_ROOT%{_docdir}/%{name}-%{version}/ \
 
 %files devel
 %defattr(-, root, root)
-
 %{_libdir}/lib*.so
-#needed to build python
-%{_libdir}/*.sh
 %{_includedir}/*
 %{_bindir}/xml2-config
 %{_datadir}/aclocal/libxml.m4
 %{_libdir}/pkgconfig/libxml-2.0.pc
 %{_libdir}/cmake/%{name}/%{name}-config.cmake
-
-%files python-build
-%defattr(-, root, root,-)
-%{_libdir}/libxml2.la
-
-%files doc
-%defattr(-, root, root)
-%doc %{_docdir}/%{name}-%{version}/*
-%doc %{_mandir}/man1/xml2-config.1*
-%doc %{_mandir}/man3/libxml.3*
+%exclude %{_libdir}/*.la
```

Comment:
the *.la file should be gone without this, that should be a bug in our rpm packaging.

Reply:
What do you mean a bug? That hasn't been include in the main package earlier, the subpackage isn't afaik used by anything, and on many packages this just gets deleted.

Reply:
The rpm packaging scripts that run after the package should cleanup the *.la files.